.github/workflows/rc_branch.yaml

```diff
@@ -0,0 +1,82 @@
+name: Workflow for 'rc*' branch
+
+# When someone comment "/deploy" on anyone PR,
+# this workflow will be triggered and run.
+on:
+  push:
+    branches:
+      - 'rc/*'
+      
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full  
+jobs:  
+  checkout: 
+    name: ckb-checkout  
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: make build    
+          - os:  mac-runner
+            script_run: export PATH="/Users/administrator/.cargo/bin:$PATH" && make build 
+          - os: win-runner
+            script_run: devtools/windows/make prod 
+    steps:
+    - name: Set env
+      run: echo "CARGO_TARGET_DIR=$(echo $HOME/target)" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+        fetch-depth: 2
+        ref: ${{ github.event.pull_request.head.sha }}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
+    - run: ${{ matrix.script_run }}
+  Benchmarks_Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+        -  Linux-runner
+        -  mac-runner
+    needs: checkout
+    steps: 
+    - name: Set env
+      run: echo "CARGO_TARGET_DIR=$(echo $HOME/target)" >> $GITHUB_ENV
+    - name: Benchmarks_Test
+      run: |
+        if [ ${{ runner.os }} == "macOS" ]; then
+        export PATH="/Users/administrator/.cargo/bin:$PATH"
+        fi
+        make bench-test
+  Integration_Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: RUST_BACKTRACE=1 RUST_LOG=${INTEGRATION_RUST_LOG} test/run.sh --release -- --bin $CARGO_TARGET_DIR/release/ckb ${CKB_TEST_ARGS}  
+          - os:  mac-runner
+            script_run: export PATH="/Users/administrator/.cargo/bin:$PATH" && RUST_BACKTRACE=1 RUST_LOG=${INTEGRATION_RUST_LOG} test/run.sh --release -- --bin $CARGO_TARGET_DIR/release/ckb ${CKB_TEST_ARGS}
+          - os: win-runner
+            script_run: devtools/windows/make CKB_TEST_SEC_COEFFICIENT=5 CKB_TEST_ARGS="-c 4 --no-report" integration 
+            CI: true
+            SENTRY_DSN: 'https://15373165fbf2439b99ba46684dfbcb12@sentry.nervos.org/7'
+            LOGBAK_USER: $(LOGBAK_USER)
```

Comment:
That's just my confusion.

Could you tell me what `$(LOGBAK_*)` means?
Have you ever tested this job for "win-runner"?

Reply:
I am not sure what $(LOGBAK_*) means, it's the env var from azure. The integration test passed on win-runner, but I am trying to find away to make sure I can get value via this way.

Reply:
it's seems uploda log.zip to azure

Reply:
Need keep those vars, will upload the log file when the test failed. but i have not test that part yet